### ISSUE TITLE
ZeroPad2D on GeometricTensor

### ISSUE THREAD

**Issue by surajp92** [QUVA-Lab/e2cnn#50]
Is it possible to do ZeroPad2D on the geometric tensor? Specifically, I want my input image and output image to have the same dimension and I am using kernel size = 4. With the normal CNN, I can achieve this with **pad = torch.nn.ZeroPad2d((2, 1, 2, 1))**. How can I do this type of padding with equivariant CNNs?
